docs/changes.xml

```diff
@@ -31,6 +31,12 @@ NGINX Unit updated to 1.27.0.
          date="" time=""
          packager="Andrei Belov &lt;defan@nginx.com&gt;">
 
+<change type="feature">
+<para>
+supporting variables in the "location" URI option in the "return" action.
```

Comment:
supporting variables in the "location" URI option AND in the "return" action.

Reply:
But "location" belongs to "return", so the AND is confusing. I suggest:
```supporting variables in the "location" option of the "return" action.```

Reply:
I like @lcrilly 's line.